sdks/java/build-tools/src/main/resources/beam/findbugs-filter.xml

```diff
@@ -0,0 +1,479 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is to reduce the possibility of conflicts when we refer to the resource. To consume this file, we declare a dependency on the `build-tools` module and refer to it as `beam/findbugs-filter.xml`. If it was at the root, we would refer to `findbugs-filter.xml`, which could also exist from another dependency.

This is part of the "Multimodule Configuration" template we used from checkstyle described [here](https://maven.apache.org/plugins/maven-checkstyle-plugin/examples/multi-module-config.html).